src/models/Popup.js

```diff
@@ -243,7 +282,7 @@ export default class Popup extends Base {
    * @return { array<boolean> } - true if consent has been given
    */
   hasConsented() {
-    return this.getStatuses().map( status => status === statusAllow || status === statusDismiss )
+    return this.getStatuses().map(status => status === STATUS_ALLOW || status === STATUS_DISMISS)
```

Comment:
Is the dismiss an implicit consent?! #LEGAL 🤓

Reply:
this part of code is not even used anywhere - not my code but we can change to ALLOW only or remove this method